editor/js/Strings.js

```diff
@@ -317,7 +317,7 @@ function Strings( config ) {
 			'sidebar/project/renderer': 'Renderer',
 			'sidebar/project/antialias': 'Antialias',
 			'sidebar/project/shadows': 'Shadows',
-			'sidebar/project/physicallyCorrectLights': 'Physical lights',
+			'sidebar/project/useLegacyLights': 'Physical lights',
```

Comment:
Shouldn't we rename the string? I think user wouldn't expect that "Physical lights: false" enables physical lights and "true" disables.

Reply:
Updated.